stm32cube/stm32l5xx/drivers/include/stm32l5xx_hal_def.h

```diff
@@ -59,9 +59,9 @@ typedef enum
 
 /* Exported macros -----------------------------------------------------------*/
 
-#ifndef UNUSED
-  #define UNUSED(X) (void)X      /* To avoid gcc/g++ warnings */
-#endif
+#if !defined(UNUSED)
+#define UNUSED(X) (void)X      /* To avoid gcc/g++ warnings */
+#endif /* UNUSED */
```

Comment:
Are we sure this is the correct fix vs README description ?
If confirmed, clear the patch list

Reply:
Yes, it is correct because "#ifndef" and "#if !defined" are equivalent